fadecandy_driver/scripts/fadecandy_node

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env python
-
```

Comment:
I think it is good to keep the python driver in the package and simply rename the executable. Just as a fallback in case we require the python driver version at some point. What do you think?

Reply:
In my opinion it does not make sense to keep the python version if the `cpp` version works correctly. The git repo will still keep the source in history if these are desired at some point.